ast2vec/repo2/base.py

```diff
@@ -126,22 +127,41 @@ def thread_loop(thread_index):
                             break
                         try:
                             folder, filename, language = task
-                            filepath = os.path.join(folder, filename).encode('utf8')
+                            filepath_naive = os.path.join(folder, filename).encode('utf8')
```

Comment:
For example [this repo](https://github.com/ROBelgium/MSNoise) has bad filename:
https://github.com/ROBelgium/MSNoise/blob/master/msnoise/unused/%C3%A903_prewhitened.py
And If I remove `.encode('utf8')` I have
```
ERROR:repo2source:Error while processing ('/tmp/repo2-r27femkp/ROBelgium@MSNoise', 'msnoise/unused/é03_prewhitened.py', 'Python')
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/ast2vec/repo2/base.py", line 86, in thread_loop
    islink = os.path.islink(filepath_naive)
  File "/usr/lib/python3.5/posixpath.py", line 161, in islink
    st = os.lstat(path)
UnicodeEncodeError: 'ascii' codec can't encode character '\xe9' in position 53: ordinal not in range(128)
```
And I have this problem only under Ubuntu, not MacOS.
Os function can not deal with Unicode string (may be because of locality or whatever) and the simplest way to avoid this problem use byte sequence.

I check it one time again. So, I should keep it.
If you want I can make str filepath and byte filepath just to avoid call .decode('utf8') each time in logging.